### DISCUSSION TITLE
RuntimeError: The size of tensor a (89) must match the size of tensor b (64) at non-singleton dimension 3

### DISCUSSION THREAD

**Discussion by fload** [hlky/stable-diffusion#230] [Q&A]
Does the Img to Img expect a certain image size to work? having trouble getting it to work with any image, just starting to investigate but it seems like the image is supposed to be sized by the front-end before it gets pushed to the backend i suspect theres a disconnect somewhere given this error...

python3.8/site-packages/anyio/_backends/_asyncio.py", line 867, in run
    result = context.run(func, *args)
  File "scripts/webui.py", line 1288, in img2img
    output_images, seed, info, stats = process_images(
  File "scripts/webui.py", line 819, in process_images
    samples_ddim = func_sample(init_data=init_data, x=x, conditioning=c, unconditional_conditioning=uc, sampler_name=sampler_name)
  File "scripts/webui.py", line 1194, in sample
    xi = x0 + noise
RuntimeError: The size of tensor a (128) must match the size of tensor b (64) at non-singleton dimension 3

if i change the image size manually i get a different tensor size....
ran git pull --update , and now getting this error
result = context.run(func, *args)
TypeError: img2img() missing 1 required positional argument: 'fp'

**Comment by altryne** [comment]
Please post an issue with this 🙏  
https://github.com/hlky/stable-diffusion-webui/issues
